content/en/boilerplates/before-you-begin-egress.md

```diff
@@ -9,19 +9,19 @@
     [configuration profile](/docs/setup/additional-setup/config-profiles/).
     {{< /tip >}}
 
-*   Deploy the [sleep]({{< github_tree >}}/samples/sleep) sample app to use as a test source for sending requests.
+*   Deploy the [curl]({{< github_tree >}}/samples/curl) sample app to use as a test source for sending requests.
     If you have
     [automatic sidecar injection](/docs/setup/additional-setup/sidecar-injection/#automatic-sidecar-injection)
     enabled, run the following command to deploy the sample app:
 
     {{< text bash >}}
-    $ kubectl apply -f @samples/sleep/sleep.yaml@
+    $ kubectl apply -f @samples/curl/curl.yaml@
     {{< /text >}}
 
-    Otherwise, manually inject the sidecar before deploying the `sleep` application with the following command:
+    Otherwise, manually inject the sidecar before deploying the `curl` application with the following command:
 
     {{< text bash >}}
-    $ kubectl apply -f <(istioctl kube-inject -f @samples/sleep/sleep.yaml@)
+    $ kubectl apply -f <(istioctl kube-inject -f @samples/curl/curl.yaml@)
```

Comment:
way off topic from this, but I thought we removed all kube-inject at some point?

Reply:
https://github.com/istio/istio.io/issues/15777